src/components/DetailsCard/CardDesktop.tsx

```diff
@@ -0,0 +1,204 @@
+      padding: 0,
+    },
+    paddingTop: theme.spacing(11),
+  },
+  avatar: {
+    position: 'absolute',
+    top: 0,
+    left: '50%',
+    marginLeft: theme.spacing(-12.5),
+    width: 200,
+    height: 200,
+    borderRadius: 16,
+    boxShadow: '0px 10px 40px -10px rgba(0,0,0,0.2)',
+  },
+  content: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    padding: 0,
+  },
+  basicInfo: {
+    background: 'transparent',
+    height: '100%',
+  },
+  moreInfo: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'flex-start',
+    background: theme.palette.background.light,
+    padding: theme.spacing(3),
+    minHeight: 268,
+    width: '100%',
+  },
+  detailsItem: {
+    marginBottom: theme.spacing(2),
+  },
+  detailsItemAvailable: {
+    color: theme.palette.success.main,
+  },
+  name: {
+    paddingTop: theme.spacing(2),
+    textAlign: 'center',
+    wordWrap: 'break-word',
+  },
+  position: {
+    color: theme.palette.primary.main,
+    marginTop: theme.spacing(0.75),
+    textAlign: 'center',
+  },
+  link: {
+    width: 21,
+    height: 21,
+    margin: theme.spacing(0, 1),
+  },
+  iconsWrapper: {
+    display: 'flex',
+    width: '100%',
+    justifyContent: 'center',
+    flexDirection: 'row',
+    alignItems: 'center',
+    marginTop: theme.spacing(2.5),
+  },
+  icon: {
+    boxSizing: 'border-box',
+    background: theme.palette.text.secondary,
+    fill: theme.palette.text.secondary,
+    stroke: theme.palette.text.white,
+    width: 21,
+    height: 21,
+    borderRadius: 3,
+    padding: 3,
+  },
+  buttonWrapper: {
+    display: 'flex',
+    width: '100%',
+    justifyContent: 'center',
+    paddingTop: theme.spacing(1.5),
+  },
+  button: {
+    width: 158,
+    height: 40,
+    borderRadius: 8,
+    boxShadow: '0 2px 4px 0 rgba(47, 84, 235, 0.15), 0 8px 16px 0 rgba(47, 84, 235, 0.15)',
+  },
+  resetAnchorStyle: {
+    textDecoration: 'none',
+    color: 'unset',
+  },
+}));
+
+const CardDesktop: FC<CardProps> = ({
+  fullName,
+  image,
+  position,
+  socialMedia,
+  phone,
+  email,
+  location,
+  isFreelancer = false,
+  resumeLink,
+}) => {
+  const classes = useStyles();
+  const displayLocation = location && `${location.city}, ${location.country}`;
+
+  return (
+    <Box className={classes.wrapper}>
+      <Card className={classes.card}>
+        <CardMedia className={classes.avatar} image={image} />
+        <CardContent className={classes.content}>
+          <Box className={classes.basicInfo}>
+            <Box className={classes.name}>
+              <Typography variant="h1">{fullName.firstName}</Typography>
+              <Typography variant="h1">{fullName.lastName}</Typography>
+            </Box>
+            <Typography variant="subtitle1" className={classes.position}>
+              {position}
+            </Typography>
+            <Box className={classes.iconsWrapper}>
+              {socialMedia?.map((media) => (
```

Comment:
Do we need the Box from line 149 if `socialMedia` prop is undefined?

Reply:
Probably not, but will check and if so, I will prevent it from rendering when undefined 👌

Reply:
Changed ✅